### DISCUSSION TITLE
ERROR: Caught unexpected exception while performing operation 'LazyInitializationOperation'!

### DISCUSSION THREAD

**Discussion by vroosioks** [SafeExamBrowser/seb-win-refactoring#592] [Q&A]
We use the SEB configuration file to set up the exam. One school had a problem when it started SEB - all students received a message: failed to start a new session.

Students pass the test in the school's computer class, all computers are in the school domain. The teacher tried to open the same test on his personal computer, which is not in the domain, and everything worked.

I add one student log files.
[2023-03-29_14h07m51s_Client.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/11100903/2023-03-29_14h07m51s_Client.log)
[2023-03-29_14h07m51s_Runtime.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/11100904/2023-03-29_14h07m51s_Runtime.log)

**Comment by dbuechel** [comment]
SEB hasn't been correctly installed on these systems, or rather, they have installed the setup (.msi) instead of the setup bundle (.exe) and are thus missing the required runtime dependencies. So either install them manually or uninstall SEB and then reinstall it using the setup bundle (.exe).

**Reply by vroosioks** [reply]
They installed SEB again in school and solved the problem. 
Thank you for your help!

**Comment by sahil3105kumar** [comment]
it the issue still persists, make sure u have given all the permissions like location and all. if not working yet, then try to disable antivirus.

**Comment by pierr3301** [comment]
i had a critical error on my SEB 3.10 version . just i had a problem with my SEB . before i download 3.10 version , i already have 3.6 and 3.8 version . i tried with my SEB 3.8 version but it works , except 3.10 version

[2025-10-02_18h47m27s_Client.log](https://github.com/user-attachments/files/22658873/2025-10-02_18h47m27s_Client.log)

[2025-10-02_18h47m27s_Runtime.log](https://github.com/user-attachments/files/22658874/2025-10-02_18h47m27s_Runtime.log)

**Reply by dbuechel** [reply]
Make sure you have all the required runtime dependencies installed as specified in the release notes (or use the setup bundle for installation, which should install them automatically).

**Comment by xinkailin-BCC** [comment]
Mine shows this...any solution?


Caught unexpected exception while performing operation 'ConfigurationOperation'!

   Exception Message: Method not found: 'Void SafeExamBrowser.Settings.Proctoring.ProctoringSettings.set_ForceRaiseHandMessage(Boolean)'.
   Exception Type: System.MissingMethodException
